manifests/3.0.0/opensearch-3.0.0.yml

```diff
@@ -6,7 +6,7 @@ build:
 ci:
   image:
     name: opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2
-    args: -e JAVA_HOME=/opt/java/openjdk-20
+    args: -e JAVA_HOME=/opt/java/openjdk-17
```

Comment:
```suggestion
    args: -e JAVA_HOME=/opt/java/openjdk-19
```

Reply:
Yeah went through old PR and just reverted what it was before

Reply:
Thank you